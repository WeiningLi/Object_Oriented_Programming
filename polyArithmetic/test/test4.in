create a 0/1 7 0/1 6 0/2 5 0/4 5 0/5 3 0/9 1 0/1 0
print a 
create b 0/5 10 1/10 7 0/1 6 0/2 5 0/4 5 0/5 3 0/9 1 0/1 0
print b
create c 0/1 0
print c
subtract a b
add a b
add a c
add c a
subtract a c
subtract c a
multiply a b
multiply b a
multiply a c
multiply c a
multiply b c
multiply c b
divide a b
divide c b
move_assign b a
print a
move_copy a c
print c
