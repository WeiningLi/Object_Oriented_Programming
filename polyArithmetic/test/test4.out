0
(1/10)x^7
0
(-1/10)x^7
(1/10)x^7
0
0
0
0
0
0
0
0
0
0
Quotient: 0
Remainder: 0
Quotient: 0
Remainder: 0
(1/10)x^7
(1/10)x^7
(1/10)x^7
(1/10)x^7
